cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -37,8 +37,8 @@
 from cirq_google import transformers
 from cirq_google.api import v2
 from cirq_google.devices import known_devices
-from cirq_google.devices.coupler import Coupler
 from cirq_google.experimental import ops as experimental_ops
+from cirq_google.ops.coupler import Coupler
```

Comment:
For the sake of consistency with `ops.Stuff` syntax at 49-52, can we drop this import and use `ops.Coupler` below?

Reply:
Added as cg_ops (it's cirq_google.ops)